sharpy/postproc/writevariablestime.py

```diff
@@ -169,20 +169,37 @@ def initialise(self, data, custom_settings=None):
 
     def run(self, online=False):
 
+        if online:
+            self.data = self.write(-1)
+        else:
+            for it in range(len(self.data.structure.timestep_info)):
+                if self.data.structure.timestep_info[it] is not None:
+                    self.data = self.write(it)
+
+        return self.data
+
+    def write(self, it):
+
         # FoR variables
         if 'FoR_number' in self.settings:
             pass
         else:
             self.settings['FoR_number'] = np.array([0], dtype=int)
 
+        if self.data.structure.timestep_info[it].in_global_AFoR:
+            tstep = self.data.structure.timestep_info[it]
+        else:
+            tstep = self.data.structure.timestep_info[it].copy()
+            tstep.whole_structure_to_global_AFoR(self.data.structure)
```

Comment:
Would you mind adding to the docs that any output from `WriteVariablesTime` is in `A` FoR?

Reply:
This piece of code does not mean that the output is going to be on the A FoR. All the postprocessors output the same variables as in the single body cases. In the case of "WriteVariablesTime" the variables are output in the same FoR in which they are stored. For example, the positions are in A FoR but the forces are in B FoR.

The issue with the multibody case is that we use several A FoR (one per body) and these lines of code just move everything to the same A FoR (named "global_AFoR") to have a coherent output.

That said, it is true that I always need to think twice about which FoR are variables referred to and it is also a source of problems for new users. Thus, I might add it somewhere in the documentation so it is clear from now on.

Reply:
Ok, makes sense. Many thanks again!